app/services/webhook_sender.go

```diff
@@ -33,7 +37,7 @@ func retry(schedule []time.Duration, fn func() error) error {
 	return err
 }
 
-func WebhookSender(destination *url.URL, values *url.Values, schedule []time.Duration) error {
+func WebhookSender(destination *url.URL, values *url.Values, schedule []time.Duration, key []byte) error {
```

Comment:
is it better to pass in a `hash.Hash` here to better support extension?  EG a function on config builds it based on the key (and eventually alg etc) specified in configuration?